### ISSUE TITLE
Files are not indexed automatically

### ISSUE THREAD

**Issue by Openhabber** [nextcloud/files_fulltextsearch#205]

**Comment by Openhabber** [comment]
thanks for your answer.
Yes, it is activated. Also php occ fulltextsearch:live does NOT pick up the files.

Only php occ fulltextsearch:index seems to pick it up.
Maybe it has something to do with it being an external storage of type "LOCAL"?

![image](https://user-images.githubusercontent.com/56535530/198856528-d34b359e-d9f3-42e3-890b-ca358c599c2e.png)

**Comment by Openhabber** [comment]
Just an addition (this is NOT a fix, but a workaround until it is fixed)
I added a cronjob on my host, which triggers indexing every night:
`0 2 * * * docker exec -it -u 33 nextcloud-aio-nextcloud php occ fulltextsearch:index -n -q 2>&1`

this is for sure not optimal, but at least it works in some way...

im happy to give more infos!

**Comment by Zoey2936** [comment]
@szaimen I think the problem is that AIO needs to run the indexing in cron or on a similar way

**Comment by szaimen** [comment]
No, I talked with @ArtificialOwl about this and he said that a cronjob is automatically scheduled by the app. So this sounds like an issue with the app together with external storage...

**Comment by Openhabber** [comment]
Any update on this? Full text search is quite an important feature...
